src/utils/convertCSVToJSON.ts

```diff
@@ -41,6 +42,7 @@ export const convertCSVToJSON: (string) => Promise<Collection[]> = (
                 credit,
                 price_guidance: price_guidance ? Number(price_guidance) : null,
                 show_on_editorial: Boolean(show_on_editorial),
+                is_featured_artist_content: Boolean(show_on_editorial),
```

Comment:
why are we using `show_on_editorial` here?

Reply:
Ah! Copy-paste, will update this now